api/src/main/java/org/hawkular/inventory/api/Relationships.java

```diff
@@ -91,14 +117,17 @@ private Relationships() {
     /**
      * Provides read-write access to relationships.
      */
-    public interface ReadWrite extends ReadWriteInterface<Relationship, Relationship.Blueprint, Single, Multiple> {
```

Comment:
For the impossibility of commenting on unchanged lines, I comment here but really want to talk about the `Single` interface.

It provides access to single relationship and its "other ends". I am not sure the `*.ReadRelate` is correct interface for accessing the other ends, because the relationship is already established and a single relationship cannot have multiple sources or targets.